core/templates/dev/head/domain/summary/ExplorationSummaryBackendApiService.ts

```diff
@@ -50,6 +50,12 @@ oppia.factory('ExplorationSummaryBackendApiService', [
       }).then(function(response) {
         var summaries = angular.copy(response.data.summaries);
         if (successCallback) {
+          if (!summaries) {
+            var summariesError = (
+              'Summaries fetched are null for explorationIds: ' + explorationIds
+            );
+            throw summariesError;
```

Comment:
You should generally be throwing an Error in JS, not a string. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error#Examples for examples.

Reply:
Done.